File: amadeus.backend/src/Services/amadeus.backend.Products/Domain/Product.cs
namespace Domain
{
    public class Product : BaseEntity
    {
        public string BarCode {  get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public decimal Price { get; set; }
        public uint StockQuantity { get; set; }
    }
}

File: amadeus.frontend/src/app/core/entities/Product.ts
export interface Product {
  barCode: string
  name: string
  description: string
  price: number
  stockQuantity: number
  id: string
  tmStmp: string
}


export interface AddProductPetition {
  barCode: string
  name: string
  description: string
  price: number
  stockQuantity: number
}
